Week-1/how-web-works/how-web-works.md

```diff
@@ -0,0 +1,116 @@
+## What is the main functionality of a browser ? ##
+
+A browser uses a URI (Uniform Resource Identifier) and sends requests for data(HTML/CSS/JS/XML/JSON and others) to other nodes(server/another computer) in its network and serve received data to the requested user.
```

Comment:
What is its network ?

Reply:
A browser can request files served by a server in local network (intranet) or the internet. I made the distinction between them. Hence I chose the word "network".